OWNERS.md

```diff
@@ -0,0 +1,18 @@
+# Senior committers
+
+When possible, all PRs should be reviewed by at least one senior committer prior to merging.
+
+* Matt Klein (mattklein123) (mklein@lyft.com)
+* Harvey Tuch (htuch) (htuch@google.com) (pending)
+
+# Committers
+
+* Roman Dzhabarov (RomanDzhabarov) (rdzhabarov@lyft.com)
+* Constance Caramanolis (ccaraman) (ccaramanolis@lyft.com)
+* Jose Nino (junr03) (jnino@lyft.com)
+* Daniel Hochman (danielhochman) (dhochman@lyft.com)
+* Michael Schore (goaway) (mschore@lyft.com)
+
```

Comment:
Yes, I have thoughts on this, both for committers from an existing org (Google/Lyft), and from an entirely new org. I'm going to type up more text on this over the weekend and we can review in the next draft iteration.